app/forms/withdrawal_form.rb

```diff
@@ -14,6 +14,7 @@ def commencement_date
   def save!
     if valid?
       assign_attributes_to_trainee
+      Trainees::Update.call(trainee: trainee, update_dtq: false)
```

Comment:
I could try and send the `withdraw_date` specifically in the params here like with the `award_date` in the `RecommendForAwardJob` but I thought it might not be necessary

Reply:
Also haven't included a spec for this - wasn't sure how to implement so would be good to get thoughts

Reply:
You're right, there's no need to explicitly pass in params. The `withdraw_date` (plus `withdraw_reason` and `additional_withdraw_reason`) will be set on the trainee during `assign_attributes_to_trainee` and saved in the next line `Trainees::Update` since that service saves the trainee.

Reply:
Since you're now calling the `Trainee::Update` service, can you remove the `trainee.save!` from the `Trainees::Withdraw` service? I think you're saving the trainee twice now.

Reply:
You could test this in the `WithdrawalForm` spec - expect that a call to `Trainees::Update` is made on `#save!` - very much the same as you've done in `recommend_for_award_job_spec`!